recipes/openexr/3.x/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "3.3.2":
+    url: "https://github.com/AcademySoftwareFoundation/openexr/archive/refs/tags/v3.3.2.tar.gz"
```

Comment:
Please prefer to use `https://github.com/AcademySoftwareFoundation/openexr/releases/download/v3.3.2/openexr-3.3.2.tar.gz`  as that is guaranteed to be hash-stable (see https://github.com/orgs/community/discussions/46034)

Also note that since this release was published about ~2 hours ago - we will wait 24-48 hours before merging - at the scale of Conan Center - we see that during the first day or two that's when it is most likely that a maintainer will re-tag/re-call/re-publish a version, invalidating the checksum (unless you are the maintainer and can advise better).

Reply:
Thanks! :)